<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Brainstorm</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/images/Brainstorm_128x128x32.png</normaloff>:/icons/images/Brainstorm_128x128x32.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="margin">
     <number>3</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitterNote">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitterLists">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QListView" name="listCategories">
        <property name="showDropIndicator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="modelColumn">
         <number>0</number>
        </property>
       </widget>
       <widget class="QListView" name="listNotes">
        <property name="maximumSize">
         <size>
          <width>16777214</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QComboBox" name="comboColors">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFontComboBox" name="comboFonts">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Calibri</family>
             </font>
            </property>
            <property name="editable">
             <bool>false</bool>
            </property>
            <property name="currentFont">
             <font>
              <family>Calibri</family>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboFontSizes">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>5.5</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>6</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>6.5</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>7</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>8</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>9</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>10</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>11</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>12</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>13</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>14</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>15</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>16</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>18</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>20</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>22</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>24</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>26</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>28</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>30</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>32</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>36</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>42</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextBrowser" name="textNote">
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="autoFormatting">
           <set>QTextEdit::AutoNone</set>
          </property>
          <property name="tabChangesFocus">
           <bool>false</bool>
          </property>
          <property name="undoRedoEnabled">
           <bool>true</bool>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextEditable|Qt::TextEditorInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
          </property>
          <property name="openExternalLinks">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>749</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Category"/>
    <addaction name="actionRename_Category"/>
    <addaction name="actionDelete_Category"/>
    <addaction name="separator"/>
    <addaction name="actionNew_Note"/>
    <addaction name="actionRename_Note"/>
    <addaction name="actionSave_Note"/>
    <addaction name="actionDelete_Note"/>
    <addaction name="separator"/>
    <addaction name="actionImport_Notes"/>
    <addaction name="actionExport_Category"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionFind_Replace"/>
   </widget>
   <widget class="QMenu" name="menuFormat">
    <property name="title">
     <string>Format</string>
    </property>
    <addaction name="actionBold"/>
    <addaction name="actionItalic"/>
    <addaction name="actionUnderline"/>
    <addaction name="actionStrikethrough"/>
    <addaction name="separator"/>
    <addaction name="actionBullet_List"/>
    <addaction name="actionNumber_List"/>
    <addaction name="separator"/>
    <addaction name="actionIncrease_Indent"/>
    <addaction name="actionDecrease_Indent"/>
    <addaction name="separator"/>
    <addaction name="actionShow_Fonts"/>
    <addaction name="actionShow_Colors"/>
    <addaction name="actionIncrease_Font"/>
    <addaction name="actionDecrease_Font"/>
    <addaction name="actionReset_Font"/>
    <addaction name="separator"/>
    <addaction name="actionAlign_Left"/>
    <addaction name="actionAlign_Center"/>
    <addaction name="actionAlign_Right"/>
    <addaction name="actionAlign_Justify"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionHide_Window"/>
    <addaction name="separator"/>
    <addaction name="actionPrevious_Category"/>
    <addaction name="actionNext_Category"/>
    <addaction name="separator"/>
    <addaction name="actionPrevious_Note"/>
    <addaction name="actionNext_Note"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_Qt"/>
    <addaction name="actionAbout_Brainstorm"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuFormat"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionBold"/>
   <addaction name="actionItalic"/>
   <addaction name="actionUnderline"/>
   <addaction name="actionStrikethrough"/>
   <addaction name="separator"/>
   <addaction name="actionAlign_Left"/>
   <addaction name="actionAlign_Center"/>
   <addaction name="actionAlign_Right"/>
   <addaction name="actionAlign_Justify"/>
   <addaction name="separator"/>
   <addaction name="actionBullet_List"/>
   <addaction name="actionNumber_List"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionNew_Category">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/toolbar/images/folder-new.png</normaloff>:/toolbar/images/folder-new.png</iconset>
   </property>
   <property name="text">
    <string>New Category</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+N</string>
   </property>
  </action>
  <action name="actionRename_Category">
   <property name="text">
    <string>Rename Category</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+R</string>
   </property>
  </action>
  <action name="actionDelete_Category">
   <property name="text">
    <string>Delete Category</string>
   </property>
  </action>
  <action name="actionNew_Note">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/toolbar/images/document-new.png</normaloff>:/toolbar/images/document-new.png</iconset>
   </property>
   <property name="text">
    <string>New Note</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionRename_Note">
   <property name="text">
    <string>Rename Note</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionDelete_Note">
   <property name="text">
    <string>Delete Note</string>
   </property>
  </action>
  <action name="actionSave_Note">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/toolbar/images/document-save.png</normaloff>:/toolbar/images/document-save.png</iconset>
   </property>
   <property name="text">
    <string>Save Note</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionImport_Notes">
   <property name="text">
    <string>Import Notes...</string>
   </property>
  </action>
  <action name="actionExport_Category">
   <property name="text">
    <string>Export Category</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/toolbar/images/edit-undo.png</normaloff>:/toolbar/images/edit-undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/toolbar/images/edit-redo.png</normaloff>:/toolbar/images/edit-redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/toolbar/images/edit-cut.png</normaloff>:/toolbar/images/edit-cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/toolbar/images/edit-copy.png</normaloff>:/toolbar/images/edit-copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionFind_Replace">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/toolbar/images/edit-find-replace.png</normaloff>:/toolbar/images/edit-find-replace.png</iconset>
   </property>
   <property name="text">
    <string>Find &amp;&amp; Replace</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionBold">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/toolbar/images/format-text-bold.png</normaloff>:/toolbar/images/format-text-bold.png</iconset>
   </property>
   <property name="text">
    <string>Bold</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionItalic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/toolbar/images/format-text-italic.png</normaloff>:/toolbar/images/format-text-italic.png</iconset>
   </property>
   <property name="text">
    <string>Italic</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionUnderline">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/toolbar/images/format-text-underline.png</normaloff>:/toolbar/images/format-text-underline.png</iconset>
   </property>
   <property name="text">
    <string>Underline</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionStrikethrough">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/toolbar/images/format-text-strikethrough.png</normaloff>:/toolbar/images/format-text-strikethrough.png</iconset>
   </property>
   <property name="text">
    <string>Strikethrough</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+K</string>
   </property>
  </action>
  <action name="actionList">
   <property name="text">
    <string>List</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionShow_Fonts">
   <property name="text">
    <string>Show Fonts</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionShow_Colors">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/toolbar/images/format-font-color.png</normaloff>:/toolbar/images/format-font-color.png</iconset>
   </property>
   <property name="text">
    <string>Show Colors</string>
   </property>
  </action>
  <action name="actionIncrease_Font">
   <property name="text">
    <string>Increase Font</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
  </action>
  <action name="actionDecrease_Font">
   <property name="text">
    <string>Decrease Font</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionReset_Font">
   <property name="text">
    <string>Reset Font</string>
   </property>
  </action>
  <action name="actionAlign_Left">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/toolbar/images/format-justify-left.png</normaloff>:/toolbar/images/format-justify-left.png</iconset>
   </property>
   <property name="text">
    <string>Align Left</string>
   </property>
  </action>
  <action name="actionAlign_Center">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/toolbar/images/format-justify-center.png</normaloff>:/toolbar/images/format-justify-center.png</iconset>
   </property>
   <property name="text">
    <string>Align Center</string>
   </property>
  </action>
  <action name="actionAlign_Right">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/toolbar/images/format-justify-right.png</normaloff>:/toolbar/images/format-justify-right.png</iconset>
   </property>
   <property name="text">
    <string>Align Right</string>
   </property>
  </action>
  <action name="actionHide_Window">
   <property name="text">
    <string>Hide Window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionPrevious_Category">
   <property name="text">
    <string>Previous Category</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Up</string>
   </property>
  </action>
  <action name="actionNext_Category">
   <property name="text">
    <string>Next Category</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Down</string>
   </property>
  </action>
  <action name="actionPrevious_Note">
   <property name="text">
    <string>Previous Note</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Up</string>
   </property>
  </action>
  <action name="actionNext_Note">
   <property name="text">
    <string>Next Note</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Down</string>
   </property>
  </action>
  <action name="actionAbout_Brainstorm">
   <property name="text">
    <string>About Brainstorm</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutQtRole</enum>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/toolbar/images/edit-paste.png</normaloff>:/toolbar/images/edit-paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionAlign_Justify">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/toolbar/images/format-justify-fill.png</normaloff>:/toolbar/images/format-justify-fill.png</iconset>
   </property>
   <property name="text">
    <string>Align Justify</string>
   </property>
  </action>
  <action name="actionBullet_List">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/toolbar/images/list-bullets.png</normaloff>:/toolbar/images/list-bullets.png</iconset>
   </property>
   <property name="text">
    <string>Bullet List</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionNumber_List">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/toolbar/images/list-numbered.png</normaloff>:/toolbar/images/list-numbered.png</iconset>
   </property>
   <property name="text">
    <string>Number List</string>
   </property>
  </action>
  <action name="actionAlpha_List">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/toolbar/images/list-lower-alpha.png</normaloff>:/toolbar/images/list-lower-alpha.png</iconset>
   </property>
   <property name="text">
    <string>Alpha List</string>
   </property>
  </action>
  <action name="actionIncrease_Indent">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/toolbar/images/format-indent-more.png</normaloff>:/toolbar/images/format-indent-more.png</iconset>
   </property>
   <property name="text">
    <string>Increase Indent</string>
   </property>
  </action>
  <action name="actionDecrease_Indent">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/toolbar/images/format-indent-less.png</normaloff>:/toolbar/images/format-indent-less.png</iconset>
   </property>
   <property name="text">
    <string>Decrease Indent</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
